tests/testconfig.ttl

```diff
@@ -34,6 +34,9 @@
     # customize the service name
     skosmos:serviceName "Skosmos being tested" ;
     skosmos:serviceNameLong "Skosmos being tested, long title"@en, "Skosmos-long" ;
+    skosmos:serviceDescription "Description of Skosmos being tested"@en ;
```

Comment:
Yes, they should, but since this is just one piece of the larger SEO puzzle, I will add the documentation after all of the configuration settings etc. have been implemented.